<launch>

  <node pkg="line_detection" name="backproject_noskel" type="backproject-noskel.py" output="screen">
  </node>

  <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure" output="screen" />

  <node pkg="image_view" name="image_raw" type="image_view" output="screen">
    <remap from="image" to="/camera/image_raw" />
    <param name="~image_transport" value="compressed" />
  </node>

  <node pkg="image_view" name="image_view_line" type="image_view" output="screen">
    <remap from="image" to="/line_image" />
    <param name="~image_transport" value="compressed" />
  </node> 
  
</launch>